.github/workflows/gradle-check.yml

```diff
@@ -27,8 +27,28 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Get changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v44
+        with:
+          files_ignore: |
+            release-notes/*.md
+            .github/**
+            ADMINS.md
+            CHANGELOG-3.0.md
+            CHANGELOG.md
+            CODE_OF_CONDUCT.md
+            CONTRIBUTING.md
+            DEVELOPER_GUIDE.md
+            MAINTAINERS.md
+            README.md
+            RELEASING.md
+            SECURITY.md
+            TESTING.md
+            TRIAGING.md
```

Comment:
What's the reason not to include all *.md at root?

Reply:
See the conversation here https://github.com/opensearch-project/OpenSearch/pull/13498#discussion_r1586988920